script.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+const filename = process.argv[2];
+const name = process.argv[3]
+let greeting = `console.log(Hello ${name})`
```

Comment:
This is confusing. Doesn't this write "console.log" in the file? We just wanted the file to say Hello NAME